Intersect.Client/Core/Graphics.cs

```diff
@@ -151,51 +151,74 @@ public static void DrawIntro()
 
         public static void DrawMenu()
         {
-            // Draw an animated background in the main menu.
-            if (ClientConfiguration.Instance.MenuBackgroundAnimated)
+            // Animated background in the main menu.
+            if (ClientConfiguration.Instance.MenuBackground.Count > 1)
             {
-                DrawFullScreenTextureAnimated(
-                    ClientConfiguration.Instance.MenuBackground,
-                    ClientConfiguration.Instance.MenuBackgroundFrameInterval,
-                    ClientConfiguration.Instance.MenuBackgroundDisplayMode
+                sMenuBackground = sContentManager.GetTexture(
+                    TextureType.Gui, ClientConfiguration.Instance.MenuBackground[sMenuBackgroundIndex]
                 );
+
+                if (sMenuBackground == null)
+                {
+                    return;
+                }
+
+                if (sMenuBackgroundInterval < Timing.Global.Milliseconds)
```

Comment:
Can you cache `Timing.Global.Milliseconds` in a variable here instead of recalling it on L169?